src/app/storage.js

```diff
@@ -0,0 +1,97 @@
+var utils = require('./utils');
+
+function Storage (updateFiles) {
+
+  var EDITOR_SIZE_CACHE_KEY = 'editor-size-cache';
+
+  this.rename = function (originalName, newName) {
+    var content = this.get(originalName);
+    this.set(newName, content);
+    this.remove(originalName);
+  };
+
+  this.remove = function (name) {
+    window.localStorage.removeItem(name);
+  };
+
+  this.setEditorSize = function (size) {
```

Comment:
I am encapsulating the `EDITOR_SIZE_CACHE_KEY` constant in storage.js, which is where it belongs since it's only used for localStorage.

`storage.setEditorSize` and `storage.getEditorSize` are both extracted (and now called) from app.js.